### PR TITLE
Add skeleton code & infra for monitoring lambda

### PR DESCRIPTION
**Pull request by NovemberTang**
## What does this change?

- Adds CDK test snapshot
- Very simple python program to demonstrate that the lambda will run
- CI script that packages the python code
- Adds Github Action to upload artifacts to riff-raff

## How to test

Run the GHA to verify it passes, then deploy via riff-raff

## How can we measure success?

The stack is created and the lambda runs

### CODE REVIEW

File: cdk/lib/snyk-tag-monitor.ts [hunk 1]
```diff
@@ -6,31 +6,36 @@ import type { App} from "aws-cdk-lib";
...
       fileName: `${app}.zip` 
     }
 
-    const snykTagLambda = new GuScheduledLambda(this,
+    new GuScheduledLambda(this,
       app, lambdaProps)
 
-    const snykApiKey = new Secret(this, "snykApiKey")
-    const snykGroupId = new Secret(this, "snykGroupId")
+    new Secret(this, "snykApiKey")
```
Comment (NovemberTang) [inline cdk/lib/snyk-tag-monitor.ts:37]: These secrets might be changed to SSM params in a different PR

File: main.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+def hello(event, context):
```
Comment (NovemberTang) [inline main.py:2]: V simple to verify packaging, deployment and execution works. Actual application code coming in a separate PR

**Review by akash1810 — APPROVED** [review]
img
